engine/test/integration_tests/dm_case_sensitive/run.sh

```diff
@@ -12,9 +12,9 @@ TABLE_NUM=500
 
 function run() {
 	start_engine_cluster $CONFIG
-	wait_mysql_online.sh --port 3306
-	wait_mysql_online.sh --port 3307
-	wait_mysql_online.sh --port 4000
+	wait_mysql_online.sh --port 3306 --try-nums 100
```

Comment:
Can make `100` as the default value of `try-nums` in `wait_mysql_online.sh`.

Reply:
Fix it.